suite-native/app/.env.debug

```diff
@@ -1,3 +0,0 @@
-EXPO_PUBLIC_ENVIRONMENT=debug
-EXPO_PUBLIC_CODESIGN_BUILD=false
-EXPO_PUBLIC_IS_ANALYTICS_LOGGER_ENABLED=false
```

Comment:
`EXPO_PUBLIC_IS_ANALYTICS_LOGGER_ENABLED`  is now intended to use only in my local .env right?

Reply:
can be used in .env.development.local, mentioned in README